### DISCUSSION TITLE
Error in Server log

### DISCUSSION THREAD

**Discussion by sysopdm** [Ylianst/MeshCentral#5510] [Issues]
Here is a new one.  Any ideas?

-------- 11/3/2023, 8:38:21 PM ---- 1.1.15 --------

(node:1359) Warning: An error event has already been emitted on the socket. Please use the destroy method on the socket while handling a 'clientError' event.
(Use `node --trace-warnings ...` to show where the warning was created)

**Comment by si458** [comment]
Yes i have recently discovered this, it's nothing to worry about,
It's happened because we upgraded modules, so something is trying to emit the error event when it's already been emitted,
Haven't found where it's coming from yet

**Reply by sysopdm** [reply]
Sounds good and thanks.

**Reply by si458** [reply]
just added a commit to enable this feature, so if you enable it, it enables it,
it wasnt working previously! https://github.com/Ylianst/MeshCentral/commit/c286c21fb6d4ae6ac1d8c9e8e8abd7520791d618
so hopefully fingers crossed, i can now find where its coming from

**Comment by si458** [comment]
**MY NOTES**
```
-------- 11/7/2023, 3:48:15 PM ---- 1.1.15 --------

(node:397) Warning: An error event has already been emitted on the socket. Please use the destroy method on the socket while handling a 'clientError' event.
    at warnUnclosedSocket (node:_http_server:839:11)
    at TLSSocket.socketOnError (node:_http_server:853:5)
    at onParserExecuteCommon (node:_http_server:888:19)
    at onParserExecute (node:_http_server:809:3)
```
